.github/workflows/build-executor.yml

```
@@ -0,0 +1,41 @@
+      - name: Log in to Docker registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ secrets.DOCKER_REGISTRY_URL }}
+          username: ${{ secrets.DOCKER_LOGIN }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Set DEV1C_EXECUTOR_API_KEY env
+        run: echo "DEV1C_EXECUTOR_API_KEY=${{ secrets.DEV1C_EXECUTOR_API_KEY }}" >> $GITHUB_ENV
+
+      - name: Build Docker image
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)